tests/unit/editor/editor-destroy-test.js

```diff
@@ -1,48 +0,0 @@
-const { module, test } = window.QUnit;
-import Helpers from '../../test-helpers';
-import { MOBILEDOC_VERSION } from 'content-kit-editor/renderers/mobiledoc';
-
-import { Editor } from 'content-kit-editor';
-
-let editor;
-let editorElement;
-
-const mobiledoc = {
-  version: MOBILEDOC_VERSION,
-  sections: [
-    [],
-    [[
-      1, 'P', [[[], 0, 'HELLO']]
-    ]]
-  ]
-};
-
-
-module('Unit: Editor #destroy', {
-  beforeEach() {
-    let fixture = $('#qunit-fixture')[0];
-    editorElement = document.createElement('div');
-    fixture.appendChild(editorElement);
-    editor = new Editor({mobiledoc});
-    editor.render(editorElement);
-  },
-  afterEach() {
-    if (editor && !editor._isDestroyed) {
-      editor.destroy();
-    }
-  }
-});
-
-test('removes toolbar from DOM', (assert) => {
```

Comment:
it may still be worth testing that the remaining views (such as the tooltip) added to the editor get destroyed, and that event listeners are unregistered, if that is not tested elsewhere.